frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -47,14 +48,16 @@ export const ClusterServiceVersionHeader: React.SFC = () => <ListHeader>
   <ColHead className="col-lg-3 col-md-3 hidden-sm hidden-xs">Provided APIs</ColHead>
 </ListHeader>;
 
-const menuActions = [Kebab.factory.Edit, Kebab.factory.Delete];
+const menuActions = [Kebab.factory.Edit];
```

Comment:
There's been a UX desire for the kebab options to include "full delete" (identical to [**Remove Subscription...** option from `Subscription` list view](https://github.com/openshift/console/blob/master/frontend/public/components/operator-lifecycle-manager/subscription.tsx#L36)). Can we include that in this PR?

Reply:
Yeah, I wanted to add that, but it didn't look possible with what data we have right now.

Kebab factory uses this for deletion:
```
Delete: (kind, obj) => ({
    label: `Delete ${kind.label}`,
    callback: () => deleteModal({
      kind,
      resource: obj,
    }),
    accessReview: {
      group: kind.apiGroup,
      resource: kind.path,
      name: obj.metadata.name,
      namespace: obj.metadata.namespace,
      verb: 'delete',
    },
  }),
```

The deleteModal would require information like resource.metadata.name for the Subscription, which the CSV doesn't have access to. (Similar issue to the "Edit Subscription" action.)

Reply:
Yeah, I tried this and it runs into the same issue as the "Edit Subscription" kebab option - we don't know the subscription's name.

Reply:
If there's no simple way to add it now, we can make that a follow on.

Reply:
Let's make sure there's an issue tracking it, though, so we don't forget.

Reply:
It's already included in https://jira.coreos.com/browse/CONSOLE-1521.